source/EngineTests/PreviewDescriptionConverterServiceTests.cpp

```diff
@@ -81,7 +85,10 @@ TEST_F(PreviewDescriptionConverterServiceTests, convertTwoCellCreature_separated
     });
     input.addConnection(1, 2);
 
-    auto result = PreviewDescriptionConverterService::get().convert(std::move(input));
+    GenomeDescription genome;
+    genome._genes = {GeneDescription().nodes({NodeDescription().color(2), NodeDescription().color(3)})};
```

Comment:
use single chained calls for creation of a Description here and on the other changes, e.g. auto genome = GenomeDescription().genes(...).